pytorch_lightning/profiler/pytorch.py

```diff
@@ -607,40 +628,26 @@ def __init__(
             self.context_names = {}
             self.running_stack = []
             self.profiler = None
-            self.lightning_module = None
+            self.lightning_module = None  # TODO: where is this set?
             self.register = None
 
             self.output_fname = output_filename
             self.output_file = None
             if local_rank is not None:
-                super().on_train_start(local_rank=local_rank)
+                super().on_train_start(local_rank=local_rank)  # TODO: why super()?
```

Comment:
Shouldn't this call `self.on_train_start`?

Reply:
Yes, should be the same here. I overrided it and forgot to remove this part.